src/plots/polar/layout_defaults.js

```diff
@@ -113,6 +107,26 @@ function handleDefaults(contIn, contOut, coerce, opts) {
                 break;
 
             case 'angularaxis':
+                // We do not support 'true' date angular axes yet,
+                // users can still plot dates on angular axes by setting
+                // `angularaxis.type: 'category'`.
+                //
+                // Here, if a date angular axes is detected, we make
+                // all its corresponding traces invisible, so that
+                // when we do add support for data angular axes, the new
+                // behavior won't conflict with existing behavior
```

Comment:
Beautifully done. Where should we document the use of categories for displaying polar dates? Just in an example somewhere? I'm thinking that doing it really right would involve categories like "Jan 2010", "Feb 2010", ... "Jan 2011" etc., `angularaxis.period` and then `tickvals`/`ticktext` to get labels to just be "Jan", "Feb", "Mar"...

Reply:
In https://github.com/plotly/plotly.js/pull/2200/commits/8b4c2b5bd9e4303599f3a407a7261bb4c3a31708, I restricted the period for category axes to be `>=` than the number of categories, as I couldn't think of a use case for winding category labels. I don't think they make sense **unless** the users set angular `tickvals`/`ticktext`. That might not be too much to ask though :thinking: 

So, we could allow `period < ax._categories.length` to make date-like categories work with a period less than the data range or simply wait for a true official handling of angular dates. I'm not sure it's worth the trouble, but I'll leave it up to you to decide.

Reply:
I wouldn't *prohibit* `period < ax._categories.length` - someone will find a use for it for sure. As long as ticks only display for the first revolution, ideally with the ability to simplify them with `tickvals`/`ticktext`, and hovertext displays the actual angular data value so you know which revolution you've hovered on. But it's also a weird case so I'm fine with leaving this feature in a polar TODO issue.